### PR TITLE
Update eg-cr.json

### PR DESCRIPTION
**Pull request by andreaszenasidi**
Github: https://github.com/org-id/register/issues/482

### CODE REVIEW

File: lists/eg/eg-cr.json [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
-    "sector": null,
-    "url": "http://www.gafi.gov.eg/ar/onestopshop/onestop.aspx",
-    "name": {
-        "en": "General Authority for Free Zones and Investment (GAFI), Commercial Registry",
-        "local": ""
-    },
-    "coverage": [
-        "EG"
+  "name": {
+    "en": "Commercial Registry",
```
Comment (kd-ods) [inline lists/eg/eg-cr.json:3]: It's good to add the country to the title, when it isn't obvious. So: "Commercial Registry, Egypt"

File: lists/eg/eg-cr.json [hunk 2]
```diff
@@ -1,46 +1,48 @@
...
+    "local": ""
```
Comment (kd-ods) [inline lists/eg/eg-cr.json:4]: It looks like 'السجل التجاري' is Commercial Registry in Arabic. If you have any quick way of checking that (and, ideally, finding the Arabic for Egyptian Commercial Registry) then please edit this.

**Review by kd-ods — CHANGES_REQUESTED** [review]
@andreaszenasidi - one or two minor edits to make.

File: lists/eg/eg-cr.json (hunk 1 above)
Comment (andreaszenasidi) [reply lists/eg/eg-cr.json]: @kd-ods thanks for checking the new registration agency. I made the 2 required changes.

**Comment by andreaszenasidi** [comment]
@kd-ods thanks for the suggestions, I made the 2 required changes.

**Review by kd-ods — APPROVED** [review]
Thanks, @andreaszenasidi. I'll merge and deploy now.

**Pull request merged**